app/src/main/java/com/github/libretube/api/obj/Segment.kt

```diff
@@ -12,7 +12,8 @@ data class Segment(
     val segment: List<Double> = listOf(),
     val userID: String? = null,
     val videoDuration: Double? = null,
-    val votes: Int? = null
+    val votes: Int? = null,
+    var skipped: Boolean? = false
```

Comment:
Why's `skipped` nullable here? Shouldn't it always either be true or false?

Reply:
I originally set it as nullable, so it could be created from the API, but forgot to remove it later on